src/View/User/_editUserForm.html.twig

```diff
@@ -0,0 +1,41 @@
+{% block stylesheet %}
+<link rel="stylesheet" href="/assets/css/userForm.css">
+{% endblock %}
+
+{% block content %}
+<section class="formSection">
+<form method="post" enctype="multipart/form-data">
+	<h1>Edit user {{ user.nickName }}  </h1>
+	
+		<div class="image-holder" id="image-holder">
+		<img src="/../assets/images/profile/{{ user.avatar }}" alt="Avatar" class="userPic" >
+		</div>
+
+		<input type="hidden" value={{user.id}} name="id" id="id" placeholder="id" >
```

Comment:
Optional : Since you're having the same form for add and edit, maybe you could create some macro to avoid code repetition